protocol-units/settlement/mcr/setup/src/local/mod.rs

```diff
@@ -29,32 +29,37 @@ impl Local {
 		dot_movement: &DotMovement,
 		mut config: Config,
 	) -> Result<(Config, tokio::task::JoinHandle<Result<String, anyhow::Error>>), anyhow::Error> {
+		println!("local setup config {config:?}",);
 
 		let chain_id = 3073;
 		config.eth_connection.eth_chain_id = chain_id;
 
 		tracing::info!("Init Settlement local conf");
-			
+
 		//start local process and deploy smart contract.
 		//define working directory of Anvil
 		info!("Starting Anvil");
 		let mut path = dot_movement.get_path().to_path_buf();
 		path.push("anvil/mcr");
 		path.push(chain_id.to_string().clone());
-		tokio::fs::create_dir_all(&path).await.context("Failed to create Anvil directory").context("Failed to create Anvil directory")?;
+		tokio::fs::create_dir_all(&path)
+			.await
+			.context("Failed to create Anvil directory")
+			.context("Failed to create Anvil directory")?;
 		path.push("anvil.json");
 
-		let exists = tokio::fs::try_exists(&path).await.context("Failed to check if Anvil file exists")?;
+		let exists = tokio::fs::try_exists(&path)
+			.await
+			.context("Failed to check if Anvil file exists")?;
 
 		info!("Anvil path: {:?}", path);
-		info!{"Anvil exists: {:?}", exists};
+		info! {"Anvil exists: {:?}", exists};
```

Comment:
A nicer printout in one message, without debug cruft and unusual macro syntax:

```rust
		info!("Anvil path: `{}`, {}", path.display(), if exists { "exists" } else { "does not exist" });
```

Reply:
Change done.